From 0318d2f0a4bb9e65c84d284acff8587bd418d837 Mon Sep 17 00:00:00 2001
From: Daniel McCarney <daniel@binaryparadox.net>
Date: Wed, 1 Nov 2023 18:10:37 -0400
Subject: [PATCH 11/12] codecov: disable comments (#187)

Turn off codecov leaving comments on PRs based on the [doc
suggestion](https://docs.codecov.com/docs/pull-request-comments#disable-comment).

Resolves https://github.com/rustls/rcgen/issues/186
---
 codecov.yml | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

--- a/codecov.yml
+++ b/codecov.yml
@@ -1,2 +1 @@
-comment:
-  require_changes: true
+comment: false
